components/observations/uploader/uploadObservation.ts

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
```

Comment:
this code is copied over from `submitObservation`, but basically unchanged otherwise.